ROMFS/px4fmu_common/init.d/airframes/70000_atmos

```diff
@@ -5,7 +5,17 @@
 # @type Free-Flyer
 # @class Spacecraft
```

Comment:
How did these come about, and what differentiates a Free flyer?

Should we also recategorize https://docs.px4.io/main/en/airframes/airframe_reference.html#_2d-space-robot